components/Advocates/MeetTheAdvocates/AdvocatesMeetTheAdvocatesCard.vue

```diff
@@ -44,8 +44,7 @@ const props = withDefaults(defineProps<Props>(), {
   slackUsername: "",
 });
 
-// Tags on AppCard is an Array
-const formattedRegion = computed(() => (props.region ? [props.region] : []));
+const formattedRegion = computed(() => (props.region ? props.region : ""));
```

Comment:
Same, keep using the array, please

Reply:
updated here https://github.com/Qiskit/qiskit.org/pull/3442/commits/80a62b8e218f9e1fc38748f1d4625a9cf8fe141d